frontend/package.json

```diff
@@ -2,12 +2,14 @@
   "name": "capstone-proj",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:8080",
   "dependencies": {
     "@material-ui/core": "^4.10.2",
     "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "n": "^6.6.0",
```

Comment:
n?

Reply:
This is a node module that needed to be installed/used in order to switch between node versions